en/peripherals/gripper.md

```diff
@@ -0,0 +1,106 @@
+# Gripper Peripherals
+
+![High-load gripper example](../../assets/hardware/grippers/highload_gripper_example.jpg)
+
+Grippers are a commonly used mechanism for UAV delivery applications. There are multiple types of interfaces (e.g. PWM, CAN) for different products in the market.
+
+## Supported Gripper Types
+
+PX4 supports the following Gripper types, check the individual page to learn how to integrate them.
```

Comment:
What does it mean to "support" a gripper type? I think essentially it means that for a hardware-integrated gripper you can assign the gripper output. More generally it means that you can configure it to be triggered when the gripper commmand is encountered in a mission or via a command. Right? 

So the roboclaw is not supported because you can't set it up to close when a gripper command is sent. Right?

Reply:
Right, ones that we can configure to be a gripper is what we support. And since there are numerous other gripper types (e.g. EPM Gripper: https://ardupilot.org/copter/docs/common-electro-permanent-magnet-V3.html), I narrowed down the scope to a 'Servo Gripper'.

And this shouldn't depend on whether it can be used in mission (right now, gripper driver is tied to `payload_deliverer` module, but it should be a separate entity actually), and ideally every gripper should be controllable via the gripper MAVLink command.

And since the gripper command doesn't work, roboclaw isn't the supported gripper as you said!

Reply:
@junwoo091400 Thanks. So what "interface" is roboclaw? I assume it is not PWM, or it is more complicated than PWM servo? I..e I really don't want to talk about the driver per-se, more about the protocol.